onnx/test/cpp/inliner_test.cc

```diff
@@ -430,5 +431,85 @@ agraph (float[N, C] X, int32[N] expected) => (float Y)
   // Nested call to LogSoftmax should be inlined.
   ASSERT_FALSE(ContainsOp(model, "LogSoftmax"));
 }
+
+TEST(FunctionBuilder, AddInlinedCallBasic) {
+  // Test the AddInlinedCall functionality
+  GraphProto graph;
+
+  // Add input to graph
```

Comment:
Can you use the parser to create the graph? It will be shorter and more readable.